egs/wsj/s5/steps/nnet3/lstm/train.sh

```diff
@@ -646,9 +651,9 @@ while [ $x -lt $num_iters ]; do
       rm $dir/$[$x-1].mdl
     fi
   fi
+  rm $dir/cache.$x
```

Comment:
sorry I forgot it. added.

On Wed, Feb 10, 2016 at 9:16 PM, Daniel Povey <notifications@github.com>
wrote:

> In egs/wsj/s5/steps/nnet3/lstm/train.sh
> <https://github.com/kaldi-asr/kaldi/pull/493#discussion_r52557875>:
>
> > @@ -646,9 +651,9 @@ while [ $x -lt $num_iters ]; do
> >        rm $dir/$[$x-1].mdl
> >      fi
> >    fi
> > +  rm $dir/cache.$x
>
> do this 2>/dev/null, or you get a lot of warnings if you use --stage.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/493/files#r52557875>.
>



-- 
Xiao-hui